src/anyp/Uri.h

```diff
@@ -98,6 +98,13 @@ class Uri
     void path(const SBuf &p) {path_=p; touch();}
     const SBuf &path() const;
 
+    /**
+     * Merge a relative-URL path into the existing URI details.
+     *
+     * It is assumed that you have already ensured that the addition is relative.
+     */
+    void addRelativePath(const char *relUrl);
```

Comment:
This old description is rather unhelpful: Nobody would be able to guess what addRelativePath() actually does based on this description. However, I do not think we should fix this (for several reasons which I can detail if needed). Instead, let's just save valuable vertical space and avoid questions like "what is a relative addition?":

```suggestion
    /// integrate the given path into the existing URI using a complex algorithm
    void mergePath(const char *path);
```